### DISCUSSION TITLE
Do you have any plans to charge twidere for twitter?

### DISCUSSION THREAD

**Discussion by kim-duru** [TwidereProject/TwidereX-Android#578] [General]
Thank you for making a great app. 

I came here after looking into twidere. 

Not long ago, Twitter announced its API charging policy. 

Therefore, I wonder if you will put in a flat rate or a paid plan or maintain twidere for twitter even if you don't.

Many people, including me, really desperately want to use twidere longer.

**Comment by starfleeting** [comment]
I would happily pay to use Twidere again
